code/game/machinery/deployable.dm

```diff
@@ -88,7 +88,7 @@
 	balloon_alert(user, "deconstructing barricade...")
 	if(!tool.use_tool(src, user, 2 SECONDS, volume=50))
 		return
-	balloon_alert(user, "barricade deconstructed")
+	loc.balloon_alert(user, "barricade deconstructed")
```

Comment:
This was my code, but I want to make sure it doesn't happen again in the future.  Are balloon alerts lingering around after the object gets `qdel`?

Wouldn't it make sense to `qdel` all active balloon alerts for objects being deleted?  That would be the better fix I think.  I can look into this the next few days since I will have some free time.